File: 프로그래머스/lv0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int general = (hp / 5);
        int army = ((hp % 5) / 3);
        int work = (((hp % 5) % 3) / 1);
        int answer = general + army + work;
        return answer;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.js
function solution(hp) {
    let general = Math.floor(hp / 5);
    let army = Math.floor((hp % 5) / 3);
    let work = Math.floor(((hp % 5) % 3) / 1);
    let answer = general + army + work;
    return answer;
}
